### ISSUE TITLE
Feature Request: Re-add HTML output support

### ISSUE THREAD

**Issue by hwac121** [laramies/theHarvester#1272]
## Note we do not support installing theHarvester on andriod

**Feature Request**

**Describe the feature request or bug or other**
In the old version of theHarvester I could output the results into a formatted HTML file. I would REALLY like to have this feature back. I have no use for XML or JSON files AT ALL.

**Comment by NotoriousRebel** [comment]
We can look into readding this back as it should be trivial.

**Comment by curious-liberal** [comment]
Are you looking for support regarding this feature?

**Comment by L1ghtn1ng** [comment]
@curious-liberal Yes, if you want to add it in please do, the old setup was messy so if you can come up with a nice neat elegant solution for it please go a head :-)
